tests/integration/src/test/java/org/mule/test/config/ConfigurationAnnotationsTestCase.java

```diff
@@ -83,8 +83,7 @@ public void testFlowAnnotations()
         assertThat(getDocDescription(flow), is("Main flow"));
         assertThat(getSourceFile(flow), is("annotations.xml"));
         assertThat(getSourceFileLine(flow), is(10));
-        // initialState is added by the xsd with a default value
-        assertThat(getSourceElement(flow), is("<flow name=\"Bridge\" doc:name=\"Bridge flow\" initialState=\"started\">" + SystemUtils.LINE_SEPARATOR +
+        assertThat(getSourceElement(flow), is("<flow name=\"Bridge\" doc:name=\"Bridge flow\">" + SystemUtils.LINE_SEPARATOR +
```

Comment:
The attribute that was removed from this assertion is not present in the source XML but was added by the schema when the transformer was generating the DOM tree. Now that we don't use the schema to set the metadata (the schema is used by Spring default's parser) we can get the actual attributes from the tag in the source xml.